python/Makefile

```diff
@@ -69,7 +69,7 @@ clean:
 .PHONY: python-client preprocess-openapi check-openapi
 preprocess-openapi:
 	@echo "Preprocessing OpenAPI spec to make X-Wallet-Auth optional..."
-	@python ../scripts/preprocess_openapi.py ../openapi.yaml ../openapi_preprocessed.yaml
+	@python3 ../scripts/preprocess_openapi.py ../openapi.yaml ../openapi_preprocessed.yaml
```

Comment:
Let's revert this – is this required in order for you to run the script on your machine?